docs/docs/pii-management.mdx

```diff
@@ -0,0 +1,184 @@
+---
+id: pii-management
+sidebar_label: PII Management
+title: PII Management
+hide_table_of_contents: false
+---
+import RasaProLabel from "@theme/RasaProLabel";
+
+import RasaProBanner from "@theme/RasaProBanner";
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+:::info New in 3.6
+
+You can now anonymize personally identifiable information (PII) data in your logs and events streamed via Kafka event
+broker only. Continue reading to learn how to enable the anonymization of PII data.
+
+:::
+
+Management of sensitive customer data collected by assistants is a critical requirement for complying with regulations
+and managing the data securely while making it available for analysis.
+Rasa Pro 3.6 introduces the ability to anonymize PII data in your logs and events streamed via the Kafka event broker.
+
+Note that the tracker store continues to store conversation data which is not anonymized. This is required so that we
+retain a source of truth for pristine data required for non-repudiation purposes. In addition, because the tracker store
+data plays a vital role in the assistant's dialogue management at inference time, anonymizing data in flight could have
+undesirable consequences for the assistant's action predictions.
+
+## Architecture Overview
+
+The anonymization of the Rasa event is run through an anonymization pipeline at the end of the dialogue management
+action prediction and execution. The Rasa agent will invoke the anonymization pipeline during each user message
```

Comment:
> The Rasa agent will invoke the anonymization pipeline during each user message handling before saving the un-anonymized new events to the tracker store.

This line sounds a little contradicting 😕 we are running the anonymization pipeline and then saving un-anonymized events to the tracker store?

Reply:
It's not contradictory, I mention in L26-29 that the tracker store remains single source of truth for pristine unanonymized data. The pipeline only anonymizes data streamed to specific topics consumed by Kafka consumers. Let me know how I should rephrase this to be clear.

Reply:
can we be explicit about the sequence of events that happens here.
- Agent invokes the anonymization pipeline
- Anonymized events are published onto the event broker
- un-anonymized events are then saved in the tracker store.

In the architecture overview, i think have the order of what happens to the events is important. 
Also are we only anonymising the UserMessage event or also BotUttered?

Reply:
> can we be explicit about the sequence of events that happens here

Addressed: can we be explicit about the sequence of events that happens here

> are we only anonymising the UserMessage event or also BotUttered?

We are anonymizing UserUttered, BotUttered, SlotSet and EntitiesAdded, I've clarified the events in a subsection just below the Architecture Overview, although used their class type name rather than code reference name. Let me know which one would be clearer.

Reply:
I think class type is clear. makes sense. thank you. everything looks very clear to me 💯

Reply:
Yay, that's great! Thank you for your thorough review ⚡ I will wait for Lena's review on Monday before merging this.